plugins/hosts/linux/cap/nfs.rb

```diff
@@ -114,13 +90,50 @@ def self.nfs_opts_setup(folders)
 
             opts[:linux__nfs_options] << "anonuid=#{opts[:map_uid]}" if !hasuid
             opts[:linux__nfs_options] << "anongid=#{opts[:map_gid]}" if !hasgid
-            opts[:linux__nfs_options] << "fsid=#{opts[:uuid]}"
+            opts[:linux__nfs_options] << "fsid=#{opts[:fsid]}"
+            # We need to store reference to machine somewhere, `replicas`
+            # seems to be the only place it is possible and harmless
+            opts[:linux__nfs_options] << "replicas=#{opts[:uuid]}"
           end
         end
 
         def self.nfs_running?(check_command)
           system(check_command)
         end
+
+        def self.nfs_current_exports
+          exports = `sudo exportfs -v`
```

Comment:
> This runs locally on the host machine and will fail on any host that does not support the exportfs command.

We already depend on `exportfs` command with `nfs_apply_command`.

> We also have a subprocess library that we prefer to use as opposed to Ruby's backticks

Original implementation used `#system` but I needed command stdout so I didn't change it subprocess and used backticks. It can be refactored though.